### ISSUE TITLE
Flashing when loaded

### ISSUE THREAD

**Issue by gibboireland** [gflansburg/SpotifyPanel#1]
Hi there, display flashes when you load into the sim, and seems to show artwork for 1 second before flashing black, then shows it again.

**Comment by gflansburg** [comment]
The the new version that I just uploaded and had forgotten to upload several days ago.  Be sure to re-install the community files as well.

**Comment by gibboireland** [comment]
Running V1.3 which I believe is the latest? Still occuring.

**Comment by gflansburg** [comment]
I am unable to recreate any flashing issue.  Are you actually playing something from a Spotify application?  This is just a remote control, not an actual player.

**Comment by gibboireland** [comment]
Yes its playing from the Spotify app. Flashing also occurs on the 127.0.0 etc in a browser. Happy to repro or jump on Discord to show you.

**Comment by gflansburg** [comment]
What is your browser?

**Comment by gibboireland** [comment]
Chrome

**Comment by gflansburg** [comment]
Try 1.3.2

**Comment by gibboireland** [comment]
Really sorry, but still exact same issue with 1.3.2.

Heres a private video to show the exact problem which I hope helps; even tried running as admin.

https://youtu.be/7jv_hIGBTVU

**Comment by gflansburg** [comment]
Did you do a Ctrl-F5 to refresh the cached Javascript?

**Comment by gibboireland** [comment]
Tried that - went into dev console and did it there too. Still occuring. Im going to leave it there buddy - may come back to this in the future when its more stable. Thanks.

**Comment by gflansburg** [comment]
Your video makes it look like it is connecting to spotify and then disconnecting and reconnecting over and over again.  What does the Console log in Chrome look like?  No one else is having this issue that I am aware of.  All of the API calls are made to Spotify directly.  My app only serves up the HTML and handles the authentication.

**Comment by gibboireland** [comment]
Ok, well if Im the only one, dont worry about it. Happens with Edge too.

**Comment by gflansburg** [comment]
Edge is Chromium as well.  What does Firefox do?  Ideally I'd like to see the Console log in Chrome to see if it is throwing any errors.

**Comment by gflansburg** [comment]
Don't suppose you would be willing to beta test a completely reimagined version would you?

**Comment by gibboireland** [comment]
Yeah sure be happy to help. Let me know what you need!

**Comment by gflansburg** [comment]
Well, hopefully this works for you.  I've put a lot of time into this one when I should have been working on research papers for my PhD.

[https://www.gafware.com/Portals/9/FSRadioSetup-1.0.0.exe](https://www.gafware.com/Portals/9/FSRadioSetup-1.0.0.exe)

**Comment by liazkam** [comment]
@gflansburg hi
I tried installing the exe but it doesn't launch.
The installation didn't report any issues.
I suggest opening a Discord server. Hopefully we can offload some of the work so you can focus on your PhD :)

**Comment by gflansburg** [comment]
I don't have a code signing cert yet so Windows is probably blocking.  Check this page:

[https://www.kiwigeeker.com/how-to/block-non-store-apps-to-install-on-windows-10.html](https://www.kiwigeeker.com/how-to/block-non-store-apps-to-install-on-windows-10.html)

**Comment by gflansburg** [comment]
[https://discord.gg/GWmbHh32](https://discord.gg/GWmbHh32)

**Comment by gflansburg** [comment]
If that doesn't work, here is the not setup version:

[https://gafware.com:21112/FSRadio-1.0.0.zip](https://gafware.com:21112/FSRadio-1.0.0.zip)

**Comment by gibboireland** [comment]
Thanks was vary if an exe for something that should drop into the community folder. Will try this.

**Comment by gflansburg** [comment]
I ordered a code signing cert on Thursday.  Just waiting to jump through all the verification hoops.

**Comment by gflansburg** [comment]
I mean, thete is still an exe and a bunch of dlls required for this thing to work. It requires more than just community files. Community files alone isn't enough to satisfy Spotify's security. I explained in the original documentation that you cannot use iFrames for Spotify's authentication so an external application has to be used. If you are not using the external application then that could explain why are are getting this flashing as it is refreshing trying to get the authentication token that is passed to the exe. In addition, it is the exe that serves up the pages that are viewed in popup panel. If you are leery about using an exe than this newer version won't meet your requirements either.

**Comment by gibboireland** [comment]
Yep I think I am sorted and will use FSDesktop.com to cover my Spotify needs. Thanks for the communication on this and best of luck with it.

**Comment by gflansburg** [comment]
I use his FSKneeboard. He does great work. But just like my mods, it requires an exe in addition to the community files. Good luck.
